.ci/docker/common/install_linter.sh

```diff
@@ -15,8 +13,8 @@ chown -R jenkins pytorch
 
 pushd pytorch
 # Install all linter dependencies
-pip_install -r requirements.txt
-conda_run lintrunner init
+pip install -r requirements.txt
```

Comment:
Unrelated, but the linter fails if you don't have this because requirements.txt has fsspec and requirements-ci.txt doesn't.  I'm not sure why this doesn't fail in normal CI